core/src/main/resources/error/error-classes.json

```diff
@@ -415,20 +415,56 @@
   },
   "UNRESOLVED_COLUMN" : {
     "message" : [
-      "A column or function parameter with name <objectName> cannot be resolved. Did you mean one of the following? [<objectList>]"
+      "A column or function parameter with name <objectName> cannot be resolved."
     ],
+    "subClass" : {
+      "GENERIC" : {
```

Comment:
@cloud-fan @srielau Are you ok with such approach?

Reply:
Well, right now a sub error class spawns a separate doc page. So we are introducing complexity into the tooling. If we want to suppress this (?)
That being said I do not understand why we can't put in some palace holder like <none> or <empty> or <null>?
But if you like this one how about WITHOUT_SUGGESTION

Reply:
shall we simply add a new error class like `UNRESOLVED_COLUMN_WITH_NO_INPUT_COLUMNS`?

Reply:
If we prefer sub error class, `WITHOUT_SUGGESTION` and `WITH_SUGGESTION` LGTM

Reply:
Renamed.